src/main/java/com/commercetools/sync/products/utils/ProductReferenceResolutionUtils.java

```diff
@@ -46,8 +47,7 @@
    * Returns an {@link List}&lt;{@link ProductDraft}&gt; consisting of the results of applying the
    * mapping from {@link Product} to {@link ProductDraft} with considering reference resolution.
    *
-   * <table>
-   *   <caption>Mapping of Reference fields for the reference resolution</caption>
+   * <table summary="Mapping of Reference fields for the reference resolution">
```

Comment:
Please do not override the changes on master branch. Otherwise we would have issues in different parts of the code: c0cc6c41768bcde703616a1dbde4e2330d3df3c0

Reply:
3ff6579